Python 3.12 drops the imp module, and rather than implement a convoluted
replacement for imp.load_source to automatically populate the package version,
we can just add a placeholder and vsed it in the template.

--- ./setup.py.orig	2023-09-29 09:55:31.893470107 -0400
+++ ./setup.py	2023-09-29 09:56:17.655743399 -0400
@@ -19,14 +19,11 @@
 
 from setuptools import setup
 
-import imp
-_version = imp.load_source("dominate._version", "dominate/_version.py")
-
 long_description = open('README.md').read()
 
 setup(
   name    = 'dominate',
-  version = _version.__version__,
+  version = '@@VERSION@@',
   author  = 'Tom Flanagan and Jake Wharton',
   author_email = 'tom@zkpq.ca',
   license = 'LGPLv3',
